package.json

```diff
@@ -29,6 +29,7 @@
   "dependencies": {
     "bell": "^3.0.0",
     "blipp": "^2.0.1",
+    "bluebird": "^2.9.30",
```

Comment:
Also, for most mainstream cases, Bluebird Promises are faster than native V8 Promises. Pref all the way! :dash: